biolink-model.yaml

```diff
@@ -1148,7 +1177,7 @@ slots:
     in_subset:
       - translator_minimal
     mappings:
-      - WD:P2293
```

Comment:
should be a separate PR

Reply:
> 
> 
> should be a separate PR

There was a [separate jsonld cleanup PR, unprocessed](https://github.com/biolink/biolink-model/pull/471). I've just asked Deepak to process it since it is pretty straight forward. Again, I'll merge the fresh master back into this PR.